android/app/src/main/java/io/mosip/registration_client/MainActivity.java

```diff
@@ -138,11 +144,24 @@ public void configureFlutterEngine(@NonNull FlutterEngine flutterEngine) {
     }
 
     public void getUISchema(MethodChannel.Result result){
+        try{
+
+            String schemaJson = identitySchemaRepository.getSchemaJson(getApplicationContext(),
+                    identitySchemaRepository.getLatestSchemaVersion());
+            result.success(schemaJson.toString());
+
+        }catch (Exception e){
+
```

Comment:
Add a log.error here so that we dont miss this exception if it occurs

Reply:
Done